docker-compose.ccm.yaml

```diff
@@ -37,12 +48,33 @@ services:
   #     MINIO_ENDPOINT: "${HAWKINS_MINIO_ENDPOINT}"
   #     MINIO_ACCESS_KEY: "${HAWKINS_MINIO_ACCESS_KEY}"
   #     MINIO_SECRET_KEY: "${HAWKINS_MINIO_SECRET_KEY}"
-  nginx:
-    image: nginx:alpine
+  #   labels:
+  #     - traefik.enable=true
+  #     - traefik.http.routers.app-hawkins.rule=Host(`stager-hawkins.ccm.sickkids.ca`) # && PathPrefix(`/api`)
```

Comment:
I assume these should stay in anticipation for the Hawkins tenancy in the future?

Reply:
Yes, and as an example of how to add another tenant.